### ISSUE TITLE
[Question] code completion ignore:

### ISSUE THREAD

**Issue by ayoubelmhamdi** [charliermarsh/ruff-lsp#96]
I am unable to ignore repetitive suggestions in my editor `neovim` while using the `Lighthub` :bulb:  icon. I have noticed that two suggestions from Ruff keep appearing even though I have already considered them in the past. I believe this is due to my consistent usage of the :bulb:  which remains visible at all times, leading to the repetition of the same suggestions. Could you please provide a solution to this issue so that I can use my editor without any hindrances?

```lua
require('lspconfig').ruff_lsp.setup {
  on_attach = on_attach,
  init_options = {
    settings = {
      args = {
      ignore: {
         "Ruff: Organize Import",
         "Ruff: Fix ALL"
         --or
         1234,
         5678
        },
      },
    }
  }
}
```
![image](https://user-images.githubusercontent.com/34358145/233417416-4c7812b8-6e9d-4452-8cc7-4d6e602d99b2.png)

**Comment by charliermarsh** [comment]
Looks like this might be the same as #91 -- does that look like the same issue to you?

**Comment by dhruvmanila** [comment]
Yes, this is the same as #91.

Also, the way you would want to use the `args` settings it to provide the command-line arguments in the same format as mentioned in `ruff check --help`:

```lua
require('lspconfig').ruff_lsp.setup {
  on_attach = on_attach,
  init_options = {
    settings = {
      args = {
      	"--ignore=E501",
	  }
    }
  }
}
```

**Comment by ayoubelmhamdi** [comment]
@dhruvmanila  But these two suggestions don't have a code to ignore them
![rem-1](https://github.com/astral-sh/ruff-lsp/assets/34358145/9dc5d116-aab0-404f-8bac-49f391ecc161)
.

**Comment by dhruvmanila** [comment]
Yes, I think you're confusing code actions with rule codes. If you really want to filter these out unconditionally, you can use the `filter` option for `vim.lsp.buf.code_action` function (https://neovim.io/doc/user/lsp.html#lsp-buf).

**Comment by xuanhung1509** [comment]
I've just stumbled on the same issue. Although I can filter out those two messages with
```lua
require("lspconfig").ruff_lsp.setup({
  init_options = {
    settings = {
      fixAll = false,
      organizeImports = false,
    },
  },
})
```
But that also means I cannot use those two actions anymore. I'd love to have those actions implemented as an editor command (like in vscode). 

![image](https://github.com/astral-sh/ruff-lsp/assets/89293664/903df625-1c2e-42a9-8b9a-00ec894e28a3)

Hopefully this will be implemented in the future.

**Comment by ayoubelmhamdi** [comment]
yes. it works fine for me thank you very match.

**Comment by dhruvmanila** [comment]
> But that also means I cannot use those two actions anymore. I'd love to have those actions implemented as an editor command (like in vscode).

For Neovim you can define these commands similar to VSCode. Please refer to https://github.com/astral-sh/ruff-lsp/issues/295#issuecomment-1782498649. You can then use a keybinding to execute that command. Hope this helps!
